internal/surface/backend/barrier_test.go

```
@@ -0,0 +1,123 @@
+
+// adapterSidePackages returns every package on the adapter side of the
+// boundary: everything under internal/surface except the coordinator itself,
+// which legitimately owns the invocation and the binary-provenance policy.
+//
+// The list is derived rather than written down. A literal slice would let a
+// Phase 5 backend package that imports internal/launch pass the barrier by not
+// being listed — no failure, no signal — which turns the package's
+// load-bearing assertion into a convention someone has to remember.
+func adapterSidePackages(t *testing.T) []string {
+	t.Helper()
+	out, err := exec.Command("go", "list", modulePath+"/internal/surface/...").Output()
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)